tdms/tests/unit/test_numerical_derivative.cpp

```diff
@@ -2,10 +2,12 @@
  * @file test_numerical_derivative.cpp
  * @brief Tests of the numerical differentiation/FFT functions.
  */
-#include <catch2/catch_test_macros.hpp>
-#include <catch2/catch_approx.hpp>
 #include <fftw3.h>
 #include <spdlog/spdlog.h>
+
+#include <catch2/catch_approx.hpp>
+#include <catch2/catch_test_macros.hpp>
+
```

Comment:
Are these headers moving because of the new clang-format dotfile?

Reply:
I imagine so - I didn't touch them but VSCode is meant to do some auto-formatting on file close.
I thought
```c++
#include <foo.bar>
```
without a space after the hash was the usual standard?

Reply:
> without a space after the hash was the usual standard?

Yeah. GSG doesn't have spaces.